### ISSUE TITLE
0.8.1 release breaks xo-web minification

### ISSUE THREAD

**Issue by chewi** [vatesfr/xo-lib#21]
No idea why but changing xo-web's package.json xo-lib entry to exactly `0.8.0` makes it work again.

```
[17:09:12] 'buildScripts' errored after 1.05 min
[17:09:12] GulpUglifyError: unable to minify JavaScript
Caused by: SyntaxError: Unexpected character '`' (line: 165949, col: 105, pos: 5455624)

Error
    at new JS_Parse_Error (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:1545:18)
    at js_error (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:1553:11)
    at parse_error (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:1677:9)
    at Object.next_token [as input] (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:1946:9)
    at next (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:2075:25)
    at expect_token (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:2108:20)
    at expect (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:2113:36)
    at expr_list (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:2636:44)
    at subscripts (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:2775:30)
    at subscripts (eval at <anonymous> (/var/www/xo-web-stable/node_modules/gulp-uglify/node_modules/uglify-js/tools/node.js:1:0), <anonymous>:2752:20)
```

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/nodejs', '/usr/bin/npm', 'run', 'build' ]
2 info using npm@3.10.8
3 info using node@v6.9.1
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle xo-web@5.3.2~prebuild: xo-web@5.3.2
6 silly lifecycle xo-web@5.3.2~prebuild: no script for prebuild, continuing
7 info lifecycle xo-web@5.3.2~build: xo-web@5.3.2
8 verbose lifecycle xo-web@5.3.2~build: unsafe-perm in lifecycle true
9 verbose lifecycle xo-web@5.3.2~build: PATH: /usr/lib/node_modules/npm/bin/node-gyp-bin:/var/www/xo-web-stable/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
10 verbose lifecycle xo-web@5.3.2~build: CWD: /var/www/xo-web-stable
11 silly lifecycle xo-web@5.3.2~build: Args: [ '-c',
11 silly lifecycle   'npm run build-indexes && NODE_ENV=production gulp build' ]
12 silly lifecycle xo-web@5.3.2~build: Returned: code: 1  signal: null
13 info lifecycle xo-web@5.3.2~build: Failed to exec build script
14 verbose stack Error: xo-web@5.3.2 build: `npm run build-indexes && NODE_ENV=production gulp build`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:877:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid xo-web@5.3.2
16 verbose cwd /var/www/xo-web-stable
17 error Linux 3.16.0-4-amd64
18 error argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "build"
19 error node v6.9.1
20 error npm  v3.10.8
21 error code ELIFECYCLE
22 error xo-web@5.3.2 build: `npm run build-indexes && NODE_ENV=production gulp build`
22 error Exit status 1
23 error Failed at the xo-web@5.3.2 build script 'npm run build-indexes && NODE_ENV=production gulp build'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the xo-web package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     npm run build-indexes && NODE_ENV=production gulp build
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs xo-web
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls xo-web
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by julien-f** [comment]
Thx for you report, I'm fixing this ASAP!

**Comment by julien-f** [comment]
Should be fixed in 0.8.2.
